include/account/address.h

```diff
@@ -17,6 +17,8 @@ class Address
 
     explicit Address(std::string const &bech32Address);
 
+    Address& operator = (const Address &address);
```

Comment:
Shouldn't an `Address` object be immutable? Why copy another `Address` into it? Useful for initialization? Then why not use a copy constructor? (sorry if I misunderstood).

Perhaps in the future we can override the comparison operator `==`, instead? (quite useful for SDK clients IMHO).

Reply:
That's actually a very good comment. I think I needed it before and left it there.
I think you are right, address objects should be immutable, I will implement comparison operator instead